frontend/src/app/reducers/browser.js

```diff
@@ -5,9 +5,11 @@ import { isMinFirefoxVersion } from '../lib/utils';
 const userAgent = navigator.userAgent.toLowerCase();
 const isUserAgentFirefox = userAgent.indexOf('firefox') > -1;
 const isUserAgentMinFirefox = isMinFirefoxVersion(isUserAgentFirefox, userAgent, config.minFirefoxVersion);
+const locale = (navigator.language || '').split('-')[0];
```

Comment:
Could we do this with a glob or pattern or regex? Would be nice if we didn't have to change everything if/when we decide to mind locales in the future